src/main/java/com/griefcraft/lwc/LWC.java

```diff
@@ -976,34 +1005,9 @@ public void sendLocale(CommandSender sender, String key, Object... args) {
      */
     public void sendLocaleToActionBar(CommandSender sender, String key, Object... args) {
```

Comment:
Same issue in this method. Also, any particular reason why we are not prefixing action bar messages?